src/shared-components/Recurrence/RecurrenceWeeklySection.tsx

```
@@ -0,0 +1,67 @@
+> = ({ frequency, byDay, onDayClick, onWeekdayKeyDown, t }) => {
+  if (frequency !== Frequency.WEEKLY) {
+    return null;
+  }
+
+  return (
+    <div className="mb-4">
+      <span className="fw-semibold text-secondary">{t('repeatsOn')}</span>
+      <br />
+      <div
+        className="mx-2 mt-3 d-flex gap-1"
+        role="group"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<fieldset>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)